ldm/dream/pngwriter.py

```diff
@@ -88,10 +88,11 @@ def make_grid(self, image_list, rows=None, cols=None):
         height = image_list[0].height
 
         grid_img = Image.new('RGB', (width * cols, height * rows))
+        i = 0
         for r in range(0, rows):
             for c in range(0, cols):
-                i = r * rows + c
                 grid_img.paste(image_list[i], (c * width, r * height))
```

Comment:
per @blessedcoolant's comment:
```suggestion
                if i >= image_cnt:
                    break
                grid_img.paste(image_list[i], (c * width, r * height))
```

If you have 5 images, it will try to make a 2x3 grid, which works fine but you need to leave the last space blank.